Dependencies.txt

```diff
@@ -0,0 +1,9 @@
+Chatter uses EasyMock, which requires the inclusion of certain external JARs on the project classpath.
+
+Here are the required dependencies:
+EasyMock: https://bintray.com/artifact/download/easymock/distributions/easymock-3.3.1.zip
+Objenesis: https://bintray.com/artifact/download/easymock/distributions/objenesis-2.2-bin.zip
+CGLib: https://github.com/cglib/cglib/releases/download/RELEASE_3_1/cglib-3.1.jar and https://github.com/cglib/cglib/releases/download/RELEASE_3_1/cglib-nodep-3.1.jar
+ASM: http://forge.ow2.org/project/download.php?group_id=23&file_id=20550
```

Comment:
Awesome. Then you'll just need to add to the dependencies section of pom.xml (in the same manner as the JUnit dependency):
```
<dependency>
	<groupId>org.easymock</groupId>
	<artifactId>easymock</artifactId>
	<version>3.3.1</version>
</dependency>
```
After, Maven will manage the dependency for builds - no more manually adding JAR files to $PATH :bomb: :boom:
Should be able to remove the previously added JAR files and still have ```mvn test``` succeed as I did locally.